tableauserverclient/server/endpoint/workbooks_endpoint.py

```diff
@@ -236,8 +236,7 @@ def delete_permission(self, item, capability_item):
     @parameter_added_in(connections='2.8')
     def publish(
         self, workbook_item, file_path, mode,
-        connection_credentials=None, connections=None, as_job=False,
-        hidden_views=None
```

Comment:
I think the back compat answer here is that if you include hidden_views as a publish parameter, it will override anything already set on the workbook model.

Reply:
And the embedding API has isHidden as an attribute on each sheet. Given the choice, we should move the models closer together rather than farther apart, so that seems like the right design here as well.